src/Compiler/Driver/CompilerConfig.fs

```diff
@@ -815,6 +817,7 @@ type TcConfigBuilder =
                     DumpGraph = false
                 }
             dumpSignatureData = false
+            strictIndentation = None
```

Comment:
New idea (feel free to ignore):

If we conditioned the default based on `FSharpExperimentalFeaturesEnabledAutomatically` , this would automatically be turned on when the "experimental CI leg" is being run on the compiler codebase itself.

Not sure how many violations this would report ...